apps/web/src/components/templates/notification-setting-form/TemplatePreference.tsx

```diff
@@ -0,0 +1,38 @@
+import { Grid, InputWrapper } from '@mantine/core';
```

Comment:
A small suggestion, maybe worth adding a cypress component test for this one

Reply:
A good suggestion I will look into it. :)